File: Angular 10 - .NET 5/Modulo 5 - Fundamentos de ASP.NET Core/fin/front-end/src/app/peliculas/pelicula.ts
export interface PeliculaCreacionDTO{
    titulo: string;
    resumen: string;
    enCines: boolean;
    fechaLanzamiento: Date;
    trailer: string;
    poster: File;
}

export interface PeliculaDTO{
    titulo: string;
    resumen: string;
    enCines: boolean;
    fechaLanzamiento: Date;
    trailer: string
    poster: string
}

File: Angular 12 - .NET 9/Modulo 8 - Despliegues/PeliculasAPI/PeliculasAPI/Entidades/Pelicula.cs
using Microsoft.EntityFrameworkCore;
using System.ComponentModel.DataAnnotations;

namespace PeliculasAPI.Entidades
{
    public class Pelicula: IId
    {
        public int Id { get; set; }
        [Required]
        [StringLength(300)]
        public required string Titulo { get; set; }
        public string? Trailer { get; set; }
        public DateTime FechaLanzamiento { get; set; }
        [Unicode(false)]
        public string? Poster { get; set; }
        public List<PeliculaGenero> PeliculasGeneros { get; set; } = new List<PeliculaGenero>();
        public List<PeliculaCine> PeliculasCines { get; set; } = new List<PeliculaCine>();
        public List<PeliculaActor> PeliculasActores { get; set; } = new List<PeliculaActor>();
    }
}
